content/docs/rfcs/16/README.md

```diff
@@ -210,7 +218,7 @@ The `get_waku_v2_store_v1_messages` method retrieves historical messages on spec
 
 | Field | Type | Inclusion | Description |
 | ----: | :---: | :---: |----------- |
-| `topics` | `Array`[`String`] | mandatory | Array of content topics to query for historical messages |
+| `contentFilters` | `Array`[[`ContentFilter`](#contentfilter)] | mandatory | Array of content filters to query for historical messages |
```

Comment:
How come this is mandatory? One could imagine a naive query for historical messages not filtering for content, I think

Reply:
I agree, though I'd suggest first changing this in the underlying Nim API where a populated query is assumed.

Reply:
> How come this is mandatory? One could imagine a naive query for historical messages not filtering for content, I think

Right, I think also making it optional makes sense.
Opened an issue  re this https://github.com/status-im/nim-waku/issues/505